tephra-hbase-compat-0.96/src/main/java/co/cask/tephra/hbase96/coprocessor/TransactionProcessor.java

```diff
@@ -149,10 +149,13 @@ public void preGetOp(ObserverContext<RegionCoprocessorEnvironment> e, Get get, L
     throws IOException {
     Transaction tx = getFromOperation(get);
     if (tx != null) {
+      boolean excludeCurrentWritePtr = isAttributeSet(get, TxConstants.TX_EXCLUDE_CURRENT_WRITE);
```

Comment:
This simply supports the ability for clients to exclude the current write pointer from reads on a per-request basis.  I don't see where the TransactionAware needs to know whether or not a checkpoint has been performed.